static/src/javascripts/test/spec/common/commercial/commercial-feature-policies.spec.js

```diff
@@ -158,7 +158,7 @@ describe('Commercial features', ()=> {
 
     describe('Adfree experience policy', ()=> {
         it('enabling adfree hides some commercial content', ()=> {
-            userAdPreference.hideAds = true;
+            userFeatures.isAdfree = ()=> true;
```

Comment:
We might have to remove jspm in the short term so I would rather not use ES6 any more than we already are in the tests right now, as we might have to manually downgrade to ES5.